Assets/Scripts/Controllers/TileSpriteController.cs

```diff
@@ -75,8 +75,27 @@ protected override void OnChanged(Tile tile)
             Debug.ULogErrorChannel("TileSpriteController", "tileGameObjectMap's returned GameObject is null -- did you forget to add the tile to the dictionary? Or maybe forget to unregister a callback?");
             return;
         }
-        
-        tile_go.GetComponent<SpriteRenderer>().sprite = SpriteManager.GetSprite("Tile", tile.Type.Name);
+
+        // TODO Evaluate this criteria and naming schema!
+        if (DoesTileSpriteExist(tile.Type.Name + "_Heavy") && (tile.walkCount >= 30))
+        {
+            if (tile.ForceTileUpdate || tile.walkCount == 30)
+            {
+                ChangeTileSprite(tile_go, tile.Type.Name + "_Heavy");
+            }
+        }
+        else if (DoesTileSpriteExist(tile.Type.Name + "_Low") && (tile.walkCount >= 10))
```

Comment:
I'm not a fan of the 2 state only tile visualisation
What about using something like the fancy new: #1296 ?
Or something more flexible, defined in the tile type? for instance an array of threshold/sprite mapping